lib/puppetlabs_spec_helper/rake_tasks.rb

```diff
@@ -149,7 +149,7 @@ def fixtures(category)
     fixture_defaults = {}
   end
 
-  fixtures = fixtures['fixtures']
+  fixtures = fixtures['fixtures'] || {}
```

Comment:
An alternative approach would be to make the fix a little higher up when `.fixtures.yml` fails to load (due to being missing / malformed). If you do have a `.fixtures.yml` but its missing a `fixtures:` section, that seems like it should fail?

```ruby
begin
    fixtures = (YAML.load_file(ENV['FIXTURES_YML'] || fixtures_yaml) || { fixtures: {} })
  rescue Errno::ENOENT
    fixtures = { 'fixtures' => {} }
  rescue Psych::SyntaxError => e
    abort("Found malformed YAML in #{fixtures_yaml} on line #{e.line} column #{e.column}: #{e.problem}")
  end
```

Reply:
👍